pkg/flavor/kubernetes/nfs.go

```diff
@@ -1161,3 +1133,18 @@ func (flavor *Flavor) waitForDeployment(deploymentName string, nfsNamespace stri
 func int32toPtr(i int32) *int32 {
 	return &i
 }
+
+func (flavor *Flavor) getResourceQuantity(scParams map[string]string, paramKey string, defaultVal string) (resource.Quantity, error) {
+
+	if val, ok := scParams[paramKey]; ok {
```

Comment:
Suggestion:
if you think, this can increase the readability

if val, ok := scParams[paramKey]; ok {
		quantityVal = val // User defined
} else {
                quantityVal = defaultVal //Default 
}

Reply:
Thanks, fixed!